.changelog/17755.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+mesh: Stop jwt providers referenced by intentions from being deleted.
```

Comment:
I've been out of the loop for a bit; is `mesh` a commonly used prefix now?

Reply:
We saw that `mesh:` was used in some prior changelog entries, so we used it for JWT and Permissive mTLS. I assumed that `mesh:` was a replacement for `connect:` (since we've moved away from "Connect" toward "Service Mesh") but it also looks like `connect:` is still being used, so 🤷